packages/nimble-components/src/table/models/table-update-tracker.ts

```diff
@@ -118,9 +118,9 @@ export class TableUpdateTracker<
         ) {
             this.track('columnDefinition');
         } else if (
-            isColumnProperty(changedColumnProperty, 'sortingDisabled')
```

Comment:
no action needed (maybe a tech debt if it seems like a good direction): huh... half-thought but I think I like the idea of moving towards the table only relying on columnInternals and not the public api of columns. Seems like we are getting close.

Reply:
I think we are getting close. I don't have a strong opinion at this point.

Reply:
Feels nice academically, but I can't think of a concrete reason to do it yet, so no action needed if no strong opinions 👍